pandas/core/series.py

```diff
@@ -3604,11 +3604,6 @@ def swaplevel(self, i=-2, j=-1, copy=True):
         -------
         Series
             Series with levels swapped in MultiIndex.
-
-        .. versionchanged:: 0.18.1
-
-           The indexes ``i`` and ``j`` are now optional, and default to
-           the two innermost levels of the index.
```

Comment:
would it make sense to add these two lines to the parameters section?

Reply:
I've made a update.